README.md

```diff
@@ -552,9 +552,15 @@ function _e() {}
 
 Ensures jsdoc block exist
 
-Type: `Boolean` or `String`
-
-Values: `true` or `"exceptExports"` (skip `module.exports = function () {};`)
```

Comment:
We usually storing deprecated values. Like that: https://github.com/jscs-dev/node-jscs/pull/1683/files#diff-850f77639a8b2662d1628faca25a1c98R9